Documentation/network/bgp-control-plane-failure-scenarios.rst

```diff
@@ -0,0 +1,62 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _bgp_control_plane_troubeshooting:
+
+BGP Control Plane Failure Scenarios
+===================================
+
+This document describes common failure scenarios that you may encounter when
+using the BGP Control Plane and provides guidance on how to mitigate them.
+
+Cilium Agent Down
+-----------------
+
+If the Cilium agent goes down, the BGP session will be lost because the BGP
+speaker is integrated within the Cilium agent. The BGP session will be restored
+once the Cilium agent is restarted. However, while the Cilium agent is down,
+the advertised routes will be removed from the BGP peer. As a result, you may
+temporarily lose connectivity to the Pods or Services.
+
+Mitigation
+~~~~~~~~~~
+
+The recommended way to address this issue is by enabling the
+:ref:`bgp_control_plane_graceful_restart` feature. This feature allows the BGP
+peer to retain routes for a specific period of time after the BGP session is
+lost. Since the datapath remains active even when the agent is down, this will
+prevent the loss of connectivity to the Pods or Services.
+
+When you can't use BGP Graceful Restart, you can take following actions depending
+on what kind of the route you are using:
+
+.. _BGP Graceful Restart: 
```

Comment:
What is this label for? I can't see it referenced anywhere

Reply:
Thanks, deleted.